nextflow_schema.json

```diff
@@ -11,79 +11,61 @@
                     "description": "Skip genome assembly and assembly variant calling",
-                    "fa_icon": "fas fa-fast-forward",
-                    "default": false
+                    "fa_icon": "fas fa-fast-forward"
                 },
                 "skip_mapping_wf": {
                     "type": "boolean",
                     "description": "Skip read mapping (alignment)",
-                    "fa_icon": "fas fa-fast-forward",
-                    "default": false
+                    "fa_icon": "fas fa-fast-forward"
                 },
                 "skip_methylation_wf": {
                     "type": "boolean",
                     "description": "Skip generation of methylation pileups",
-                    "fa_icon": "fas fa-fast-forward",
-                    "default": false
+                    "fa_icon": "fas fa-fast-forward"
                 },
                 "skip_repeat_calling": {
                     "type": "boolean",
                     "description": "Skip tandem repeat calling",
-                    "fa_icon": "fas fa-fast-forward",
-                    "default": false
+                    "fa_icon": "fas fa-fast-forward"
                 },
                 "skip_repeat_annotation": {
                     "type": "boolean",
                     "description": "Skip tandem repeat annotation",
-                    "fa_icon": "fas fa-fast-forward",
-                    "default": false
+                    "fa_icon": "fas fa-fast-forward"
                 },
                 "skip_phasing_wf": {
                     "type": "boolean",
                     "fa_icon": "fas fa-fast-forward",
-                    "description": "Skip phasing of variants and haplotagging of reads",
-                    "default": false
+                    "description": "Skip phasing of variants and haplotagging of reads"
                 },
                 "skip_snv_annotation": {
                     "type": "boolean",
-                    "description": "Skip short variant annotation",
-                    "default": false
+                    "description": "Skip short variant annotation"
                 },
                 "skip_cnv_calling": {
                     "type": "boolean",
-                    "description": "Skip CNV calling",
-                    "default": false
+                    "description": "Skip CNV calling"
                 },
                 "skip_call_paralogs": {
                     "type": "boolean",
-                    "description": "Skip the calling of specific paralogous genes",
-                    "default": false
+                    "description": "Skip the calling of specific paralogous genes"
                 },
                 "skip_rank_variants": {
                     "type": "boolean",
-                    "description": "Skip ranking of short variants",
-                    "default": false
+                    "description": "Skip ranking of short variants"
```

Comment:
The removing is "default": false is something that we discussed at some point, its done automatically by lining isn't is?

Reply:
Yeah, it's done by `nf-core schema build` if you accidentally click yes. I will revert this.